npm-debug.log

```diff
@@ -0,0 +1,202 @@
+32 silly addNameRange     '1.2.1',
+32 silly addNameRange     '1.3.0',
+32 silly addNameRange     '1.3.1',
+32 silly addNameRange     '1.4.0',
+32 silly addNameRange     '1.4.1',
+32 silly addNameRange     '1.4.2',
+32 silly addNameRange     '1.4.3',
+32 silly addNameRange     '1.5.0',
+32 silly addNameRange     '1.5.1',
+32 silly addNameRange     '1.6.0',
+32 silly addNameRange     '1.7.0',
+32 silly addNameRange     '1.7.1',
+32 silly addNameRange     '1.7.2',
+32 silly addNameRange     '1.7.3',
+32 silly addNameRange     '1.8.0',
+32 silly addNameRange     '1.9.0',
+32 silly addNameRange     '1.10.0',
+32 silly addNameRange     '1.10.1',
+32 silly addNameRange     '1.10.2',
+32 silly addNameRange     '1.10.3' ] ]
+33 silly addNamed eslint@1.10.3
+34 verbose addNamed "1.10.3" is a plain semver version for eslint
+35 silly mapToRegistry name eslint
+36 silly mapToRegistry using default registry
+37 silly mapToRegistry registry https://registry.npmjs.org/
+38 silly mapToRegistry uri https://registry.npmjs.org/eslint
+39 verbose addRemoteTarball https://registry.npmjs.org/eslint/-/eslint-1.10.3.tgz not in flight; adding
+40 verbose addRemoteTarball [ 'https://registry.npmjs.org/eslint/-/eslint-1.10.3.tgz',
+40 verbose addRemoteTarball   'fb19a91b13c158082bbca294b17d979bc8353a0a' ]
+41 info retry fetch attempt 1 at 9:13:47 AM
+42 info attempt registry request try #1 at 9:13:47 AM
+43 http fetch GET https://registry.npmjs.org/eslint/-/eslint-1.10.3.tgz
+44 http fetch 200 https://registry.npmjs.org/eslint/-/eslint-1.10.3.tgz
+45 silly fetchAndShaCheck shasum fb19a91b13c158082bbca294b17d979bc8353a0a
+46 verbose addTmpTarball C:\cygwin64\tmp\npm-5728-8f2fd37e\registry.npmjs.org\eslint\-\eslint-1.10.3.tgz not in flight; adding
+47 verbose addTmpTarball already have metadata; skipping unpack for eslint@1.10.3
+48 silly cache afterAdd eslint@1.10.3
+49 verbose afterAdd C:\Users\Kenneth Suh\AppData\Roaming\npm-cache\eslint\1.10.3\package\package.json not in flight; writing
+50 verbose afterAdd C:\Users\Kenneth Suh\AppData\Roaming\npm-cache\eslint\1.10.3\package\package.json written
+51 verbose stack Error: ENOENT: no such file or directory, open 'C:\cygwin64\home\Kenny Suh\code-blog\[eslint-plugins]'
+51 verbose stack     at Error (native)
+52 verbose cwd C:\cygwin64\home\Kenny Suh\code-blog
+53 error Windows_NT 10.0.10240
+54 error argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "i" "--save-dev" "eslint" "[eslint-plugins]"
+55 error node v4.2.2
+56 error npm  v2.14.7
+57 error path C:\cygwin64\home\Kenny Suh\code-blog\[eslint-plugins]
+58 error code ENOENT
+59 error errno -4058
+60 error syscall open
+61 error enoent ENOENT: no such file or directory, open 'C:\cygwin64\home\Kenny Suh\code-blog\[eslint-plugins]'
+61 error enoent This is most likely not a problem with npm itself
+61 error enoent and is related to npm not being able to find a file.
+62 verbose exit [ -4058, true ]
```

Comment:
This file should be in your .gitignore. It's got some somewhat sensitive info in it like the paths to your files on your local machine, etc. You don't want an attacker to get this info. Also it isn't needed to run your app and pollutes your repo.